### DISCUSSION TITLE
zwave-js node properties

### DISCUSSION THREAD

**Discussion by fabnavigator** [zwave-js/node-red-contrib-zwave-js#192] [Q&A]
Is there documentation on all of the properties that be setup in the zwave-js node?

**Comment by marcus-j-davies** [comment]
Yes. - kind of.

Most of the properties are a direct link with the zwave-js API settings.

See here.
Some names slightly differ to make for a better understanding In node red.

Not everything is mapped - just the main areas.

https://zwave-js.github.io/node-zwave-js/#/api/driver?id=zwaveoptions

**Reply by fabnavigator** [reply]
@marcus-j-davies Thank you.
